test/unittests/commonMocks.h

```diff
@@ -161,19 +161,18 @@ class NotifierMock : public Notifier {
          * actually created/sent */
     }
 
-    MOCK_METHOD6(sendNotifyContextRequest, void(NotifyContextRequest* ncr, const std::string& url, const std::string& tenant, const std::string& xauthToken, const std::string& fiwareCorrelator, NotificationFormat notifyFormat));
+    MOCK_METHOD7(sendNotifyContextRequest, void(NotifyContextRequest* ncr, const std::string& url, const std::string& tenant, const std::string& xauthToken, const std::string& fiwareCorrelator, RenderFormat renderFormat, const std::vector<std::string>&  attrsFilter));
     MOCK_METHOD5(sendNotifyContextAvailabilityRequest, void(NotifyContextAvailabilityRequest* ncar, const std::string& url, const std::string& tenant, const std::string& fiwareCorrelator, Format f));
 
     /* Wrappers for parent methods (used in ON_CALL() defaults set in the constructor) */
-    void parent_sendNotifyContextRequest(NotifyContextRequest* ncr, const std::string& url, const std::string& tenant, const std::string& xauthToken, const std::string& fiwareCorrelator, NotificationFormat notifyFormat)
+    void parent_sendNotifyContextRequest(NotifyContextRequest* ncr, const std::string& url, const std::string& tenant, const std::string& xauthToken, const std::string& fiwareCorrelator, RenderFormat renderFormat, const std::vector<std::string>&  attrsFilter)
     {
-      return Notifier::sendNotifyContextRequest(ncr, url, tenant, xauthToken, fiwareCorrelator, notifyFormat);
```

Comment:
It is very strange ...
I did a short test just to see how it 'should' work:

XXX.cpp:

    void a(void)
    {
      return 10;
    }

    int main(int argC, char* argV[])
    {
      a();
      return 0;
    } 

This is the output from the compilation:

    kz@hoppe:fiware-orion> make XXX
    g++     XXX.cpp   -o XXX
    XXX.cpp: In function 'void a()':
    XXX.cpp:4: error: return-statement with a value, in function returning 'void'
    make: *** [XXX] Error 1

Which makes a lot of sense.
I would **not** leave it the way it was, knowing that it is incorrect.
Why doesn't the compiler complain? I have no idea. Strange things with the mocks, perhaps ...

However, I believe I found this 'error' as my compiler complained ...